app/javascript/components/ingredients/ingredients-container.vue

```diff
@@ -193,11 +193,14 @@ export default {
       this.showingAdd = !this.showingAdd;
 
       try {
+        var ingredientsInfo = this.$refs.addIngredientInfo.form;
+        ingredientsInfo.ingredient_measures_attributes = ingredientsInfo
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Identifier 'ingredient_measures_attributes' is not in camel case  camelcase